src/inspector/view/qml/MuseScore/Inspector/notation/bends/BendSettings.qml

```diff
@@ -65,9 +65,9 @@ Column {
 
     InspectorPropertyView {
         id: bendCurve
-        titleText: qsTrc("inspector", "Click to add or remove points")
+        titleText: qsTrc("inspector", "Customise bend")
```

Comment:
Customi**ze**

Reply:
Oh yeah
I see that `Customize` is also used in NoteInputBar (settings popup)
@bkunda FYI

Reply:
Sorry, that's my predilection towards British English.  
`Customize` is fine.